Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -236,16 +251,16 @@ port ``31940`` (one for each of devices ``eth0`` and ``eth1``):
 .. parsed-literal::
 
     kubectl exec -it -n kube-system cilium-fmh8d -- cilium service list
-    ID   Frontend               Service Type   Backend                    
```

Comment:
I have no idea why these changes are showing up. When I do `git annotate`, I don't see them.

Reply:
I think it's because your editor is removing the trailing spaces. It's fine.